app/presenters/timeline_events/show_presenter.rb

```diff
@@ -11,27 +11,39 @@ def page_title
     end
 
     def markdown_prop(markdown)
-      { markdown: markdown, profile: 'permissive' }.to_json
+      { markdown: markdown, profile: "permissive" }.to_json
     end
 
     def render_markdown?(item)
-      %w[longText shortText link].include?(item['kind'])
+      %w[longText shortText].include?(item["kind"])
     end
 
     def files?(item)
-      item['kind'] == 'files'
+      item["kind"] == "files"
     end
 
     def audio?(item)
-      item['kind'] == 'audio'
+      item["kind"] == "audio"
+    end
+
+    def link?(item)
+      item["kind"] == "link"
     end
 
     def audio_file(item)
-      TimelineEventFile.find(item['result'])
+      TimelineEventFile.find(item["result"])
     end
 
     def files(item)
-      TimelineEventFile.where(id: item['result'])
+      TimelineEventFile.where(id: item["result"])
+    end
+
+    def submission_from
+      if @submission.founder.team.present?
+        @submission.founder.team.name
+      else
+        @submission.founder.name
+      end
     end
```

Comment:
This logic is incorrect; 

Consider this case; The student is part of a team and the target is configured with the following config.
<img width="1038" alt="image" src="https://github.com/pupilfirst/pupilfirst/assets/14979190/12bd9a52-4622-4a4e-9b0e-dcd367acd4f4">


This function will return the team name while it has to be student's name.